docs/tutorials/tensorflow_inference_and_training.md

```diff
@@ -0,0 +1,152 @@
+# Tutorial: Optimize TensorFlow Models with BladeDISC
+
+In this tutorial, we show how to optimize TensorFlow models with BladeDISC for
+both inference and training. Users only need to add two lines of code to
+optimize the model just-in-time for the examples in this tutorial.
+
+These packages are required before going through the tour:
+
+- TensorFlow\[gpu\] (all the codes in the tutorial are verified with TF 2.4)
+- BladeDISC
+
+To build and install `BladeDISC` package, please refer to ["Install BladeDISC
+With Docker"](/docs/install_with_docker.md).
+
+
+The content of this tutorial is as following.
+- [BERT inference](#bert-inference)
+  - [Prologue: download frozen model](#prologue-download-frozen-model)
+  - [All you need are the two lines!](#all-you-need-are-the-two-lines)
+  - [Epilogue: normal process to run
+    inference](#epilogue-normal-process-to-run-inference)
+- [DeePMD training](#deepmd-training)
+  - [Prologue: install DeePMD-kit and download
+    data](#prologue-install-deepmd-kit-and-download-data)
+  - [Still, all you need are the two
+    lines!](#still-all-you-need-are-the-two-lines)
+  - [Epilogue: normal process to run MD training with DeePMD-kit
+    API](#epilogue-normal-process-to-run-md-training-with-deepmd-kit-api)
+
+
+## BERT inference
+
+BERT models are usually feed with data of dynamic shapes in real production. The
+dynamic shape mainly comes from two aspects, one is varied batch-size, and the
+other is varied data shape of each sample (e.g., varied sequence lengths). The
+model we show in this tutorial has varied batch-size.
+
+### Prologue: download frozen model
```

Comment:
Please format the title using https://titlecase.com/

Reply:
Done. Thanks.